src/gui/shader.cpp

```diff
@@ -26,7 +24,10 @@ load_shaders(
     GLuint fragment_shader_id = glCreateShader(GL_FRAGMENT_SHADER);
 
     // Read the Vertex Shader code from the file
-    LOG_INFO(logger, "Loading vertex shader from {}", vertex_file_path);
+    LOG_INFO(
+        logging::opengl_logger, "Loading vertex shader from {}",
+        vertex_file_path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.